src/plugins/saved_objects_management/public/management_section/objects_table/saved_objects_table.tsx

```diff
@@ -172,13 +179,17 @@ export class SavedObjectsTable extends Component<SavedObjectsTableProps, SavedOb
 
   componentDidMount() {
     this._isMounted = true;
+
+    this.subscribleWorkspace();
     this.fetchSavedObjects();
     this.fetchCounts();
   }
 
   componentWillUnmount() {
     this._isMounted = false;
     this.debouncedFetchObjects.cancel();
```

Comment:
Do we need to manually cancel it? is it automatically handles cleanup when the component unmounts?

Reply:
Yes, we do have unsubscribe in `componentWillUnmount` method. Refactor to a `unSubscribeWorkspace` method to make it more clear.